runtime/Swift/Antlr4/org/antlr/v4/runtime/DefaultErrorStrategy.swift

```diff
@@ -226,7 +226,11 @@ public class DefaultErrorStrategy: ANTLRErrorStrategy {
         // try cheaper subset first; might get lucky. seems to shave a wee bit off
         //let set : IntervalSet = recognizer.getATN().nextTokens(s)
 
-        if try recognizer.getATN().nextTokens(s).contains(la) || la == CommonToken.EOF {
+        if try recognizer.getATN().nextTokens(s).contains(CommonToken.EPSILON) {
```

Comment:
⚠️ This is sub-optimal but should work fine. Someone familiar with the Swift syntax should reply with a comment indicating how to cache the result of `nextTokens` in a local variable like I've done for each of the other targets.